sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -257,7 +257,14 @@ const eph = EventProcessorHost.createFromConnectionString(
 // In V2, you get a single event passed to your callback. If you had asynchronous code running in your callback,
 // it is not awaited before the callback is called for the next event.
 const onMessage = (context, event) => {
-  /** your code here **/
+  /** Your code to process the event here **/
+
+  try {
+    await context.checkpointFromEventData(event);
+  } catch (err) {
+    console.log(`Error when checkpointing on partition ${context.partitionId}: `, err);
```

Comment:
Do we **have** to show the catch, log, throw parts?

Reply:
I suppose not :smile: